wgsl/index.bs

```diff
@@ -715,8 +743,32 @@ or a [=hexadecimal floating point literal=].
   </xmp>
 </div>
 
+<div algorithm="mathematical value of hexadecimal floating point literal">
+The mathematical value of a [=hexadecimal floating point literal=] is computed as follows:
+* Compute *effective_mantissa* from *mantissa*:
+    * If *mantissa* has 17 or fewer [=mantissa/significant=] digits, then *effective_mantissa* is *mantissa*.
```

Comment:
This preserves 68 bits.  That seemed appropriate.
The decimal float literal algorithm preserves ~66.4 bits.

Reply:
Hm.  I'm not sure that there is benefit for *17* hex digits over *16* hex digits.
And using 16 allows us to keep all the bits in a uint64_t